source/io/module_javaScriptInvoke.js

```diff
@@ -289,6 +289,7 @@ var assignJavaScriptInvoke;
                 // to abort the runtime at this point. https://github.com/ni/VireoSDK/issues/521
                 throw returnValue;
             }
+            console.error(returnValue.message);
```

Comment:
Can you do `console.error(returnValue)` instead? Let the browser decide how to handle the Error object (the returnValue.message does not include the stack trace in all browsers, etc)